.generator/src/generator/templates/api.j2

```diff
@@ -300,18 +307,41 @@ impl {{ structName }} {
         {%- endif %}
         {%- endfor %}
 
+        // build headers
+        let mut headers = HeaderMap::new();
+        {%- if operation.requestBody %}
+        {%- set contentTypeHeaders = operation.requestBody.content.keys()|list %}
+        {%- if contentTypeHeaders %}
+        {%- if "application/json" in contentTypeHeaders %}
+        headers.insert("Content-Type", "application/json".parse().unwrap());
+        {%- else %}
+        headers.insert("Content-Type", "{{ contentTypeHeaders[0] }}".parse().expect("failed to parse Content-Type header"));
+        {%- endif %}
+        {%- endif %}
+        {%- endif %}
+        {%- if operation.responses %}
+        {%- set acceptHeaders = operation|accept_headers %}
+        {%- if acceptHeaders %}
+        {%- if "application/json" in acceptHeaders %}
+        headers.insert("Accept", "application/json".parse().unwrap());
+        {%- else  %}
+        headers.insert("Accept", "{{ acceptHeaders|join(",") }}".parse().expect("failed to parse Accept header"));
```

Comment:
Not a huge fan of the way this syntax is, especially since we are passing the value from client side. Could we use something like `HeaderValue::from_static("application/json")`?